adaptors/proximityadaptor-evdev/proximityadaptor-evdev.cpp

```diff
@@ -38,6 +38,7 @@
 
 #define SELFDEF_EV_SW 5
 #define SELFDEF_EV_FRONT_PROXIMITY 11
+#define SELFDEF_EV_DISTANCE 25
```

Comment:
There are ABS_DISTANCE & SW_FRONT_PROXIMITY values - is there some reason not to use them?